carsCollection.php

```diff
@@ -0,0 +1,33 @@
+<?php
+require 'functions.php';
+$getCars = getCarsDb();
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- CSS -->
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href=" css/style.css">
+    <!-- FONT -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,600;0,700;0,800;1,400&display=swap" rel="stylesheet">
+    <!-- PAGE TITLE -->
+    <title>Your Collection</title>
+</head>
+<body>
+<section>
+    <div class="button">
```

Comment:
The button (lines 21-23) is part of story 2 (remove from story1 branch, add to story1 branch)